pkg/registry/registry.go

```diff
@@ -39,7 +38,20 @@ type DriverToolkitEntry struct {
 	OSVersion           string `json:"OSVersion"`
 }
 
+func writeImageRegistryCredentials() {
+	secretNamespace := "openshift-config"
+	secretName := "pull-secret"
+	configFilePath := "/home/nonroot/.docker/config.json"
+	secretDataName := ".dockerconfigjson"
+
+	s, err := clients.Interface.CoreV1().Secrets(secretNamespace).Get(context.TODO(), secretName, metav1.GetOptions{})
+	exit.OnError(err)
```

Comment:
Instead of exit.OnError, can we log the error (missing global pull-secret) and keep trying to reconcile?

Reply:
Just realised I lost the vanilla k8s check, so I added that back. About logging and trying to reconcile, this is used from `LastLayer` function, which bails out in almost every error, so I will log and return errors and let the caller decide.